lib/electric/migrations/templates/migration.sql.eex

```diff
@@ -1,4 +1,10 @@
 /*
-ElectricDB Migration
-{"metadata": {"title": "<%= title %>", "name": "<%= name %>"}}
+Electric SQL Migration
+name: <%= name %>
+title": <%= title %>
```

Comment:
Possibly extra `"` after the title?
I wonder, why did you decide to make this data unparsable by the machine? Or is this a switch to YAML in a way?

Reply:
As this is all in the manifest now I didnt want a machine readable version in two palces